vessim/sil/node.py

```diff
@@ -21,11 +23,16 @@ class Node:
     def __init__(
         self,
         address: str,
+        name: Optional[str],
         port: int = 8000,
         power_mode: str = "high performance"
     ) -> None:
-        Node.id += 1
         self.id = Node.id
+        Node.id += 1
```

Comment:
Yes, setting the power mode in the vessim api requires a node id. This id needs to be assigned to a node somehow in the simulation. It is used for example in `vessim.cosim.sil_interface.py` to update the power mode of a node.

Reply:
Don't quite remeber why the name field exists tbh. Do we want to identify the nodes by name or id?

Reply:
I talked with philipp yesterday and we decided to omit the name and make the id a mandatory arg instead. We need the id because of the sim api. The id needs to be fixed from the start and cannot be automatically assigned. I assumed that an id always had to be an integer. The "new" id will be a string. I will also adjust this in the api.

Reply:
thats right